controllers/oscmachinetemplate_capacity_controller.go

```diff
@@ -68,6 +68,13 @@ func (r *OscMachineTemplateReconciler) Reconcile(ctx context.Context, req ctrl.R
 		Logger:             log,
 		OscMachineTemplate: machineTemplate,
 	})
+
+	if err != nil {
+		log.V(2).Info("Cluster is not available yet")
+		log.Error(err, "failed to get machineTemplate.")
+		return reconcile.Result{RequeueAfter: 30 * time.Second}, nil
+	}
+	log.V(2).WithValues("machineTemplateScope", machineTemplateScope).Info("ubble-test")
```

Comment:
Thank you for your work on this PR. I have a couple of suggestions:

**Log Message Update**: Please update the log message "ubble-test" to a more descriptive message that accurately reflects the context.

**Logging Verbosity**: Currently, the logs use log.V(2). High verbosity levels can generate excessive log data in a production environment. Could you please adjust the verbosity level to INFO or another appropriate level ?

Reply:
Sorry. this log was only for my debug. removed